lib/internal/freeze_intrinsics.js

```diff
@@ -197,7 +198,7 @@ module.exports = function() {
     // 18 The Global Object
     eval,
     isFinite,
-    isNaN,
+    NumberParseIsNaN,
```

Comment:
This has to use the global `isNaN`, since this refers to properties on the global object.

```suggestion
    // eslint-disable-next-line node-core/prefer-primordials
    isNaN,
```